fe/components/footer.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from "react";
 
 import * as styles from "@dataform/fe/components/footer.css";
 
-import FooterLogoImage from "df/static/images/new_logo_with_text_white.svg";
+const footerLogoImage = require("df/static/images/new_logo_with_text_white.svg");
```

Comment:
why are we `import`ing some SVGs and `require()`ing others?

Reply:
Oh man, I wish I knew. TLDR you have to require inside ts_library rules, and anywhere else you have to do import. Tried to work out why exactly, couldn't, gave up.